### DISCUSSION TITLE
Docusaurus template support

### DISCUSSION THREAD

**Discussion by andrey-jef** [ObsidianPublisher/obsidian-github-publisher#103] [Q&A — Template] [answered]
Hi,

The scope of the github-publisher plugin has expanded over mkdocs-material template. 

I has recently discovered [docusaurus](https://docusaurus.io/), which is also a clean and well-organized template.

Do you have interest to add the support of docusaurus template to the github-publisher plugin?

Here is an [example of a random personal blog](https://cho.sh/r), of which the author published his Obsidian vault using docusaurus template.

**Comment by Mara-Li** [comment, marked as answer]
Well, my plugin allow to send notes in any repo so Docusaurus can be supported.

But I don't think I will create plugin or code to support obsidian callout, wikilinks and other things.

**Reply by andrey-jef** [reply]
Oh. I thought the code which does the wikilink conversion and change Obsidian flavored markdown syntax to mkdocs flavor are included in the mkdocs template repo. So it's the github-publisher plugin who does both the conversion and send to repo, isn't it?

**Reply by Mara-Li** [reply]
It can be disabled but tbh I don't think it can broke links!
